.github/conda_envs/minimum-reqs.yml

```
@@ -0,0 +1,32 @@
+name: isce3
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - backoff
+  - cmake=3.19
```

Let's add a comment here explaining why this isn't `cmake=3.18`

```suggestion
  # Building pyre from source requires cmake>=3.19
  - cmake=3.19
```